include/net/net_if.h

```diff
@@ -421,7 +457,7 @@ static inline enum net_verdict net_if_recv_data(struct net_if *iface,
  * @return Return the link layer header size
  */
 static inline u16_t net_if_get_ll_reserve(struct net_if *iface,
-					  const struct in6_addr *dst_ip6)
+					 const struct in6_addr *dst_ip6)
```

Comment:
As I mentioned already earlier, this change is not related to this LL support, please remove this change.